app/assets/stylesheets/ama_layout/layout_components/banner.scss

```diff
@@ -0,0 +1,29 @@
+.banner{
+	position: relative; /* This is needed for the positioning of the close icon on the blue banners*/
+
+	&--blue{
+		background: $brand-blue-insurance;
+		padding: 20px;
```

Comment:
instead of using 20px, use $base-padding instead. It's slightly less (~16px) but it will keep the spacing consistent with other things on the page.